torch/optim/_functional.py

```diff
@@ -196,6 +198,7 @@ def adadelta(params: List[Tensor],
              grads: List[Tensor],
              square_avgs: List[Tensor],
              acc_deltas: List[Tensor],
+             foreach: bool = False,
```

Comment:
nit: why isn't this one kwarg only with the others below?
Sorry if I already asked and forgot the answer.

Reply:
ah this was because `torch/distributed` calls into this functional API and uses torchscript with it. kwarg only arguments that have a default aren't supported by torchscript so I added `foreach` before the `*` since we want the default